framework/application/application.cpp

```diff
@@ -181,16 +181,20 @@ void Application::Run()
 
 void Application::SetPaused(bool paused)
 {
-    paused_ = paused;
-
-    if (paused_ && (file_processor_ != nullptr))
+    if (file_processor_ != nullptr && paused_ != paused)
     {
         uint32_t current_frame = file_processor_->GetCurrentFrameNumber();
-        if (current_frame > 0)
+        if (paused_)
```

Comment:
Is it possible this should be `paused`?  `paused_` (with underscore) is the _previous_ value of the pause, so this would invert the messages, I think.

Reply:
Yes, this is a mistake on my side, I will fix that =)